code-challenges.test.js

```diff
@@ -23,10 +50,84 @@ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "s
 
 // b) Create the function that makes the test pass.
 
+const colors = (array) =>  {
+    return array.map(value => value.shuffle)
+}
+//--------------------------------Notes question 1---------------------------------
+//warning package.json: No license field
+// $ /Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/.bin/jest
+// console.warn
+//   YError: Invalid second argument. Expected string but received function.
+//       at c (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/yargs/build/index.cjs:1:2289)
+//       at /Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/yargs/build/index.cjs:1:2141
+//       at Array.forEach (<anonymous>)
+//       at h (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/yargs/build/index.cjs:1:2038)
+//       at Xt.describe (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/yargs/build/index.cjs:1:34403)
+//       at Object.describe (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/code-challenges.test.js:32:1)
+//       at Runtime._execModule (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/jest-runtime/build/index.js:1714:24)
+//       at Runtime._loadModule (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/jest-runtime/build/index.js:1223:12)
+//       at Runtime.requireModule (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/jest-runtime/build/index.js:1047:12)
+//       at jestAdapter (/Users/learnacademy/Desktop/week-4-assessment-VenessaSP/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapter.js:89:13)
+
+//     30 | // - Use HOF .map to iterate through each array
+//     31 |
+//   > 32 | describe ("colors", () => {
+//        | ^
+//     33 |     it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+//     34 |         const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+//     35 |         const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+
+//     at h (node_modules/yargs/build/index.cjs:1:2179)
+//     at Xt.describe (node_modules/yargs/build/index.cjs:1:34403)
+//     at Object.describe (code-challenges.test.js:32:1)
+
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+//   Your test suite must contain at least one test.
+
+//     at onResult (node_modules/@jest/core/build/TestScheduler.js:172:18)
+//     at node_modules/@jest/core/build/TestScheduler.js:300:17
+//     at node_modules/emittery/index.js:311:13
+//         at Array.map (<anonymous>)
+//     at Emittery.emit (node_modules/emittery/index.js:309:23)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       0 total
+// Snapshots:   0 total
+// Time:        0.192 s
+// Ran all test suites.
+// error Command failed with exit code 1.
+//------------------------------------------ I don't understand what I did wrong there
```

Comment:
The error you are getting is from some code that VS code added to the top of this file.  This happens from time to time, so anytime you see this error, check if there is some code that includes the word yargs, if so go ahead and remove it or comment it out.